spec/models/entity_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe Group, type: :model do
```

Comment:
- [ ] Just as requested by the previous reviewer, yu seem to not have addressed these failing specs. In your comment, if you are facing the error "pending migrations" when running the specs but you have no migrations, you can run this and it will run the pending migrations ```rails db:migrate RAILS_ENV=test```.

**You will be able to run the migrations and be able to resolve the specs that are currently failing.**